android/engine/src/main/java/org/smartregister/fhircore/engine/ui/login/LoginViewModel.kt

```diff
@@ -87,18 +97,22 @@ constructor(
     object : ResponseCallback<ResponseBody>(responseBodyHandler) {}
   }
 
+  // auth-resp (failure)
+  //   -> show error
+  //   -> attempt local login (true) -> goto home
+  // auth-resp (success)
+  //   -> fetch userinfo #LoginViewModel.responseBodyHandler
+  //   -> user-resp (failure)
+  //      -> show-error
+  //   -> user-resp (success)
+  //      -> store user info
+  //      -> goto home
```

Comment:
This is a public property so we can have the above comment in a detailed and formatted comment paragraph